### PR TITLE
Issue#129 fix styles for selector

### PR DESCRIPTION
**Pull request by ihordubas1**
#129

### CODE REVIEW

File: src/components/components/Mixins.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ export default class Mixin extends React.Component {
 
   updateMixins = (value) => {
     const entity = this.props.entity;
-
+    value = Array.isArray(value) ? value : [value];
```
Comment (kfarr) [inline src/components/components/Mixins.js]: what is this for? @ihordubas1

Comment (kfarr) [reply src/components/components/Mixins.js]: reverted

Comment (kfarr) [reply src/components/components/Mixins.js]: this looks like a way to handle a case where a non-array is passed; that shouldn't happen with the new version, reverting

Comment (ihordubas1) [reply src/components/components/Mixins.js]: @kfarr 
can you check the new PR (https://github.com/3DStreet/3dstreet-editor/pull/183) 
I think that we can close the current PR and use #183 .

**Comment by kfarr** [comment]
use https://github.com/3DStreet/3dstreet-editor/pull/183 instead

**Pull request closed without merging**